src/app/component/quiz/quiz.component.ts

```diff
@@ -20,53 +20,31 @@ import { DialogService } from 'src/app/dialog-service/dialog.service';
   styleUrls: ['./quiz.component.css'],
 })
 export class Quizcomponent implements OnInit, OnDestroy {
-  quizData: any = quizData;
+  quizData = { ...quizData };
   correctAnswer: number = 0;
   wrongAnswer: number = 0;
 
   @ViewChild('carousel')
   carousel!: NgbCarousel;
   myForm!: FormGroup;
-  name = [
-    'Q-1.',
-    'Q-2.',
-    'Q-3.',
-    'Q-4.',
-    'Q-5.',
-    'Q-6.',
-    'Q-7.',
-    'Q-8.',
-    'Q-9',
-    'Q-10',
-  ];
   question!: any;
-  options!: any;
-  title!: any;
   dialogData = { ...dialogData };
   private destroyed$: ReplaySubject<boolean> = new ReplaySubject(1);
-
-  selectedOption!: boolean;
-
   questionindex = 0;
   counter = 30;
   interval$: any;
-  validate: any;
   points: number = 0;
   correctanswer: number = 0;
   inCorrectanswer: number = 0;
-  dataSource: any;
-  startquiz: any;
-  answerlistArray: any = [];
-  submittedAnswer: any[] = [];
-  isQuizCompleted: boolean = false;
-  findIndex: any;
-  filter: any;
   filterquestionsindex: any;
-  checkIfselctedanswer: any;
-  userName: any;
   timer: any;
   selectedQuizType: any;
   radioValue: any;
+  options: string | number | undefined;
+  positivePoints: any;
```

Comment:
correctAnswer: number = 0;
  wrongAnswer: number = 0; 
  Why you have declared this  ? As in above i can see those variable

Reply:
This is uesd three time Please check 
correctanswer: number = 0;

Reply:
yes i have removed unnecessary variables